### DISCUSSION TITLE
v1.3 published

### DISCUSSION THREAD

**Discussion by WesIngwersen** [USEPA/supply-chain-factors#79] [Announcements]
Code to produce v1.3 of the Supply Chain Factors.

The associated dataset and documentation can be found at:

[Supply Chain Greenhouse Gas Emission Factors v1.3 by NAICS-6](https://doi.org/10.23719/1531143)

<hr /><em>This discussion was created from the release <a href='https://github.com/USEPA/supply-chain-factors/releases/tag/v1.3.0'>v1.3</a>.</em>

**Comment by Elmansy1231231231** [comment]
Moelmansy272@gmail.com
